### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by Nekothegamer**
Support for italian

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="drawer_home">Inizio</string>
+    <string name="drawer_archive">Archivio</string>
+    <string name="drawer_backup">Backup</string>
+    <string name="drawer_settings">Impostazioni</string>
+    <string name="drawer_rating">Valutaci</string>
+    <string name="drawer_share">Condividi</string>
+    <string name="drawer_privacy">Privacy</string>
+    <string name="drawer_footer_text">Fatto con ❤️ in India.</string>
+    <string name="drawer_share_message">
+        Hey! Dai un'occhiata a quest'app fantastica chiamata GreenStash.
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:20]: Escape (`'`) chars here, i.e. `un'occhiata` -> `un\'occhiata` same for the other one.

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -0,0 +1,224 @@
+<resources>
+    <string name="app_name">GreenStash</string>
+
+    <!-- Other Strings -->
+    <string name="confirm">Conferma</string>
+    <string name="cancel">Cancella</string>
+    <string name="ok">Ok</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:7]: Capitalize this to `OK`

**Comment by Nekothegamer** [comment]
> It seems you're having some trouble with the CI

yeah it keeps telling me there's a missing element on line 224, but i see nothing there

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="new_goal_onboarding_desc">Tocca il tasto di azione fluttuante per creare un nuovo obbiettivo di risparmio.</string>
+    <string name="home_search_label">Cerca qui…</string>
+    <string name="search_goal_not_found">404: Obbiettivo non trovato!</string>>
+
+    <!-- Deposit & Withdraw Screen -->
+    <string name="deposit_screen_title">Fantastico! Inserisci l'importo</string>
+    <string name="withdraw_screen_title">Figo! Inserisci l'importo</string>
+    <string name="deposit_successful">Depositato con successo!</string>
+    <string name="withdraw_successful">Prelevato con successo!</string>
+    <string name="withdraw_overflow_error">L'importo è di più di quanto è attualmente depositato!</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:59]: Add escaping here aswell

File: app/src/main/res/values-it/strings.xml [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="progress_greet2">Continua ad andare avanti!</string>
+    <string name="progress_greet3">Se pensi che puoi farcela, ce la fai.</string>
+    <string name="progress_greet4">Ci siamo quasi, ci sei!</string>
+    <string name="progress_greet5">Congratulazioni &#x1F389;</string>
+    <string name="currently_saved_incomplete">Attualmente hai risparmiato %s di %s.</string>
+    <string name="currently_saved_complete">Hai risparmiato %s su %s.</string>
+    <!-- Eg: You have until 25/12/2022 (152) days left. -->
+    <string name="goal_days_left">Hai fino al %s (%s) giorni rimasti.</string>
+    <!-- Eg: You need to save around $2/day, $20/week, $200/month. -->
+    <string name="goal_approx_saving">Devi risparmiare all'incirca %s/</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:92]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="goal_achieved_desc">È tempo di celebrare tutto il duro lavoro che ha portato a questa gioiosa occasione.</string>
+
+    <!-- Goal Info Screen -->
+    <string name="info_screen_header">Informazioni obbiettivo</string>
+    <string name="info_card_title">Hai Risparmiato</string>
+    <string name="info_card_remaining_amount">su %s</string>
+    <string name="info_card_remaining_days">%s giorni rimasti.</string>
+    <string name="info_card_no_deadline_set">Nessuna scadenza impostata.</string>
+    <string name="info_card_goal_achieved">Obbiettivo raggiunto! 🎉</string>
+    <string name="info_goal_priority">La priorità dell'obbiettivo è %s</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:106]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="info_copy_notes_icon_desc">Copia note</string>
+    <string name="info_copy_notes_alert">Copiato con successo!</string>
+    <string name="info_goal_no_transactions">Ancora nessuna transazione.</string>
+    <string name="info_edit_transaction_button">Aggiorna Transazione</string>
+    <string name="info_transaction_swipe_tip">Suggerimento! Scorri le transazioni a sinistra o a destra per modificarle o eliminarle.</string>
+
+    <!-- Input (New/Edit Goal) Screen -->
+    <string name="input_screen_header">È ora di risparmiare!</string>
+    <string name="input_edit_goal_header">Modifica obbiettivo di risparmio.</string>
+    <string name="input_pick_icon">Seleziona un'icona per il tuo obbiettivo.</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:120]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="input_pick_icon_onboarding_title">…O se il minimalismo è il tuo stile, aggiungi un'icona per il tuo obbiettivo.</string>
+    <string name="input_pick_icon_onboarding_desc">Le icone dell'obiettivo verranno usate sulle schede degli obbiettivi in stile compatto, Impostazioni > Stile schede obbiettivo</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:122]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="input_pick_image_fab">Immagine obbiettivo</string>
+    <string name="input_pick_image_onboarding_title">Aggiungi un'immagine al tuo obbiettivo!</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:124]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="input_pick_image_onboarding_desc">Tocca sul tasto d'azione fluttuante per aggiungere un'immagine al tuo obbiettivo.</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:125]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="input_page_quote" translatable="false">Non devi vedere tutta la scalinata, devi solo fare il primo passo. – Martin Luther King, Jr.  </string>
+    <string name="input_goal_priority">Seleziona la priorità per il tuo obbiettivo.</string>
+    <string name="input_goal_reminder">Promemoria risparmi</string>
+    <string name="input_goal_reminders_onboarding_title">Abilita i promemoria per rimanere aggiornato sui tuoi obbiettivi!</string>
+    <string name="input_goal_reminders_onboarding_desc">I promemoria verranno mandati giornalmente, semi settimanalmente, o settimanalmente, a seconda della priorità dell'obbiettivo.</string>
+    <string name="input_text_title">Titolo</string>
+    <string name="title_empty_err">Ops! Il titolo è vuoto.</string>
+    <string name="input_text_amount">Importo obbiettivo</string>
+    <string name="amount_empty_err">Ops! l'importo è vuoto o invalido.</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:134]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+
+    <!-- OSL Screen -->
+    <string name="osl_screen_header">Librerie Open Source</string>
+
+    <!-- About Screen -->
+    <string name="about_screen_header">Informazioni</string>
+    <string name="about_readme_title">README</string>
+    <string name="about_readme_desc">Vai a vedere il repository di GitHub e il README dell'app.</string>
+    <string name="about_privacy_title">Informativa sulla Privacy</string>
+    <string name="about_privacy_desc">Leggi l'informativa sulla privacy e i termini di servizio su GitHub.</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:203]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="about_support_desc">Se pensi che mi merito dei biscotti per il mio lavoro, puoi comprarne alcuni per me toccando qui.</string>
+    <string name="about_version_title">Versione</string>
+    <string name="about_version_desc">Versione-%s</string>
+
+    <!-- Widget Screen -->
+    <string name="widget_config_screen_header">Seleziona obbiettivo</string>
+    <string name="goal_widget_desc">Salvato %s</string>
+
+    <!-- Goal Reminders -->
+    <string name="notification_permission_error">Concedi l'autorizzazione alle notifiche! 😾</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:218]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="preferred_currency_setting">Valuta preferita</string>
+    <string name="currency_dialog_title">Seleziona la valuta preferita</string>
+    <string name="security_settings_title">Sicurezza</string>
+    <string name="app_lock_setting">Blocco biometrico</string>
+    <string name="app_lock_setting_desc">Usa  la tua impronta digitale o la password di blocco schermo per sbloccare GreenStash</string>
+    <string name="misc_setting_title">Miscellanee</string>
+    <string name="license_setting">Licenza &amp; Riconoscimenti</string>
+    <string name="license_setting_desc">Mostra le informazioni sulla licenza open source.</string>
+    <string name="app_info_setting">Informazioni App</string>
+    <string name="app_info_setting_desc">Mostra la versione dell'app e altri link utili.</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:189]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="info_copy_notes_alert">Copiato con successo!</string>
+    <string name="info_goal_no_transactions">Ancora nessuna transazione.</string>
+    <string name="info_edit_transaction_button">Aggiorna Transazione</string>
+    <string name="info_transaction_swipe_tip">Suggerimento! Scorri le transazioni a sinistra o a destra per modificarle o eliminarle.</string>
+
+    <!-- Input (New/Edit Goal) Screen -->
+    <string name="input_screen_header">È ora di risparmiare!</string>
+    <string name="input_edit_goal_header">Modifica obbiettivo di risparmio.</string>
+    <string name="input_pick_icon">Seleziona un'icona per il tuo obbiettivo.</string>
+    <string name="input_pick_icon_onboarding_title">…O se il minimalismo è il tuo stile, aggiungi un'icona per il tuo obbiettivo.</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:121]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="drawer_archive">Archivio</string>
+    <string name="drawer_backup">Backup</string>
+    <string name="drawer_settings">Impostazioni</string>
+    <string name="drawer_rating">Valutaci</string>
+    <string name="drawer_share">Condividi</string>
+    <string name="drawer_privacy">Privacy</string>
+    <string name="drawer_footer_text">Fatto con ❤️ in India.</string>
+    <string name="drawer_share_message">
+        Hey! Dai un'occhiata a quest'app fantastica chiamata GreenStash.
+        Rende la pianificazione e la gestione dei tuoi risparmi molto più facile, aiutandoti ad instaurare l'abitudine di risparmiare soldi per cose migliori.
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:21]: Missing escape

File: app/src/main/res/values-it/strings.xml [hunk 16]
```diff
@@ -0,0 +1,224 @@
...
+    <string name="search_button_desc">Cerca degli obbiettivi di risparmio specifici.</string>
+    <string name="new_goal_fab">Nuovo Obbiettivo</string>
+    <string name="new_goal_onboarding_title">Iniziamo a creare un nuovo obbiettivo di risparmio!</string>
+    <string name="new_goal_onboarding_desc">Tocca il tasto di azione fluttuante per creare un nuovo obbiettivo di risparmio.</string>
+    <string name="home_search_label">Cerca qui…</string>
+    <string name="search_goal_not_found">404: Obbiettivo non trovato!</string>>
+
+    <!-- Deposit & Withdraw Screen -->
+    <string name="deposit_screen_title">Fantastico! Inserisci l'importo</string>
+    <string name="withdraw_screen_title">Figo! Inserisci l'importo</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:56]: Missing escape

**Review by starry-shivam — CHANGES_REQUESTED** [review]
It seems there are quite a few unescaped `'` characters. While I've tried to mark the ones I could, I would suggest verifying it once using a dedicated XML text editor. Such editors could point out these syntactic errors. Alternatively, you can try using Android Studio to edit them, as it will automatically point out these errors in the top right corner

**Comment by Nekothegamer** [comment]
oooohh
i get it now...

**Comment by Nekothegamer** [comment]
god those are a lot of missing escapes, I'll modify all of them and upload everything as a new file

**Comment by starry-shivam** [comment]
> god those are a lot of missing escapes, I'll modify all of them and upload everything as a new file

No worries, take your time :)

File: app/src/main/res/values-it/strings.xml [hunk 17]
```diff
@@ -0,0 +1,224 @@
+<resources>
+    <string name="app_name">GreenStash</string>
+
+    <!-- Other Strings -->
+    <string name="confirm">Conferma</string>
+    <string name="cancel">Cancella</string>
+    <string name="OK">OK</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:7]: Sorry i meant to change the value of the string to `OK` from `oK` bot the name

**Comment by Nekothegamer** [comment]
nevermind still that same error...

File: app/src/main/res/values-it/strings.xml [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+    <!-- Widget Screen -->
+    <string name="widget_config_screen_header">Seleziona obbiettivo</string>
+    <string name="goal_widget_desc">Salvato %s</string>
+
+    <!-- Goal Reminders -->
+    <string name="notification_permission_error">Concedi l\'autorizzazione alle notifiche! 😾</string>
+    <string name="reminder_notification_desc">Più risparmi, più velocemente raggiungerai il tuo obbiettivo! 💰</string>
+    <string name="dismiss_notification_button">Scarta</string>
+    <string name="notification_deposited_title">Ottimo Lavoro! 🎉</string>
+    <string name="notification_deposited_desc">Depositato %s con successo!</string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:222]: You're missing closing tag here.

**Review by starry-shivam — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-it/strings.xml (hunk 17 above)
Comment (starry-shivam) [reply app/src/main/res/values-it/strings.xml]: `<string name="ok">OK</string>`

Comment (Nekothegamer) [reply app/src/main/res/values-it/strings.xml]: done

Comment (Nekothegamer) [reply app/src/main/res/values-it/strings.xml]: oh wait "ok" not "oK"? my bad

**Comment by Nekothegamer** [comment]
> @Nekothegamer i mean change it to `<string name="ok">OK</string>` the name is in smallcase while value is in upper case

oh didn't see, changing it now

**Comment by starry-shivam** [comment]
> > @Nekothegamer i mean change it to `<string name="ok">OK</string>` the name is in smallcase while value is in upper case
> 
> oh didn't see, changing it now

You've done the opposite in last commit, the name/key should be smallcass and the value of the string needs to be in upper case :)

**Comment by Nekothegamer** [comment]
> > > @Nekothegamer i mean change it to `<string name="ok">OK</string>` the name is in smallcase while value is in upper case
> > 
> > 
> > oh didn't see, changing it now
> 
> You've done the opposite in last commit, the name/key should be smallcass and the value of the string needs to be in upper case :)

oh...

so `<string name="OK">ok</string>`?

**Comment by starry-shivam** [comment]
> > > > @Nekothegamer i mean change it to `<string name="ok">OK</string>` the name is in smallcase while value is in upper case
> > > 
> > > 
> > > oh didn't see, changing it now
> > 
> > 
> > You've done the opposite in last commit, the name/key should be smallcass and the value of the string needs to be in upper case :)
> 
> oh...
> 
> so `<string name="OK">ok</string>`?

It should be ``<string name="ok">OK</string>`` from name i mean `name="ok"` while the value is the one inside `><`

**Comment by starry-shivam** [comment]
Thanks, looks like there are still a few errors left, I'll take a look in a moment

**Comment by Nekothegamer** [comment]
> Thanks, looks like there are still a few errors left, I'll take a look in a moment

yeah i have no idea what's going on now, stuff just keeps popping up and the build keeps outputting the same error, I don't know what to do

File: app/src/main/res/values-it/strings.xml [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+
+    <!-- About Screen -->
+    <string name="about_screen_header">Informazioni</string>
+    <string name="about_readme_title">README</string>
+    <string name="about_readme_desc">Vai a vedere il repository di GitHub e il README dell\'app.</string>
+    <string name="about_privacy_title">Informativa sulla Privacy</string>
+    <string name="about_privacy_desc">Leggi l\'informativa sulla privacy e i termini di servizio su GitHub.</string>
+    <string name="about_gh_issue_title">Issues di GitHub</string>
+    <string name="about_gh_issue_desc">Invia segnalazioni di bug or richieste di funzionalità creando una issue su GitHub.</string>
+    <string name="about_telegram_title">Gruppo Telegram/string>
```
Comment (starry-shivam) [inline app/src/main/res/values-it/strings.xml:206]: Missing closing tag here, Should be: ` <string name="about_telegram_title">Gruppo Telegram</string>`

**Review by starry-shivam — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-it/strings.xml (hunk 19 above)
Comment (Nekothegamer) [reply app/src/main/res/values-it/strings.xml:206]: WHAT????!? THIS WAS THE ERROR???!? oh my god i'm speechless

**Comment by starry-shivam** [comment]
Finally the CI is passing 🎉 Thanks for your patience and efforts :)

**Review by starry-shivam — APPROVED** [review]

**Comment by Nekothegamer** [comment]
yes! it's been merged!!!!! thank you as well for your immense patience and sorry if i bothered you 😅

**Comment by starry-shivam** [comment]
>  sorry if i bothered you 😅

Of course not! You've spent a significant amount of time translating those strings and contributing. In comparison, all I did was spend a few minutes pointing out some errors haha

**Comment by Nekothegamer** [comment]
> > sorry if i bothered you 😅
> 
> Of course not! You've spent a significant amount of time translating those strings and contributing. In comparison, all I did was spend a few minutes pointing out some errors haha

oh well fair enough ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

thanks for merging anyway!

**Comment by Nekothegamer** [comment]
can't wait for the translation to get released!

**Comment by starry-shivam** [comment]
> can't wait for the translation to get released!

I just created a release a few hours ago today, so the next release will take a while, probably around two weeks or so :)

**Comment by Nekothegamer** [comment]
damn, this long?

**Comment by starry-shivam** [comment]
> damn, this long?

Well, yeah, I have some other plans to do before the next release, like implementing #104, which will take some time.

**Comment by Nekothegamer** [comment]
oh, understandable

**Pull request merged**